test/test_output.sh

```diff
@@ -1,7 +1,9 @@
 #!/bin/sh
 # test/test_output.sh 
+EXIT_STATUS=0
 test $(grep -oc '"outputs": ' "$1") -eq $(grep -oc '"cell_type": "code"' "$1")
```

Comment:
This script feels like it could be simplified a bit more. For example:

```suggestion
test $(grep -oc '"outputs": ' "$1") -eq $(grep -oc '"cell_type": "code"' "$1") || exit 1
```

or instead of calling `exit 1` there, you could call a function that prints the "fail" message before calling exit 1

Reply:
where the `||` means, only run the second bit if the first bit fails

Reply:
I originally omitted this intentionally since I didn't want to fail fast, but I realize now that this script is invoked using `find` so this approach is better. Good call.